Content.Server/Abilities/Mime/MimePowersSystem.cs

```diff
@@ -28,6 +31,8 @@ public override void Initialize()
             SubscribeLocalEvent<MimePowersComponent, ComponentInit>(OnComponentInit);
             SubscribeLocalEvent<MimePowersComponent, SpeakAttemptEvent>(OnSpeakAttempt);
             SubscribeLocalEvent<MimePowersComponent, InvisibleWallActionEvent>(OnInvisibleWall);
+            SubscribeLocalEvent<MimePowersComponent, EmoteEvent>(OnEmote);
+            SubscribeLocalEvent<MimePowersComponent, ScreamActionEvent>(OnScreamAction);
```

Comment:
if this is intended to intercept the event before vocalsystem it should have a `before: new [] { typeof(VocalSystem) }` (or whatever the syntax/system name is) to make that explicit, otherwise this might randomly break

Reply:
I knew there was something like that I was forgetting. I'll update it.